CMakeLists.txt

```diff
@@ -252,167 +259,113 @@ set(AVIF_CODEC_INCLUDES)
 set(AVIF_CODEC_LIBRARIES)
 
 if(AVIF_CODEC_DAV1D)
-    message(STATUS "libavif: Codec enabled: dav1d (decode)")
```

Comment:
I moved this message to the bottom of this block, because it sounds like we've done all the checks and everything seems working.

Reply:
Moved into #816.